pkg/tcpip/stack/neighbor_cache_test.go

```diff
@@ -1750,6 +1750,34 @@ func TestNeighborCacheRetryResolution(t *testing.T) {
 	}
 }
 
+// TestNeighborCacheStaticResolution checks that static link addresses are
```

Comment:
looks like bad conflict resolution. this test is failing.

Reply:
Sorry about that. Should be fixed now.